src/pages/management/configuration-api/v2.0/index.mdx

```diff
@@ -14,7 +14,7 @@ Welcome to the LiveChat API documentation!
 
 Our API provides a set of flexible tools which you can use to create new outstanding projects. We smile a bit more each time we see skilled developers unleash their creativity!
```

Comment:
We discussed that it'd be good to include a visible warning/emoji with information about this version being deprecated

Reply:
Okay, I wrote there that it is deprecated, but a warning would definitely be better. I must have misunderstood something. I'll fix it.

Reply:
Corrected :) I also added the info about 30 days